grantnav/frontend/templates/help.html

```diff
@@ -0,0 +1,305 @@
+        </tr>
+        <tr>
+          <td>{% trans "Retrieved for use in GrantNav" %}</td>
+          <td>{% trans "This is the date and time the data was accessed for use in GrantNav" %}</td>
+        </tr>
+        <tr>
+          <td>{% trans "License (see note below)" %}</td>
+          <td>{% trans "The URL (web link) to the specific licence under which the data file containing this grant record was published." %}</td>
+        </tr>
+      </table>
+    </div>
+  </div>
+
+  <h3 id="export_funders_recipients">{% trans "Exporting from the funders and recipients pages" %}</h3>
+  <p>{% blocktrans %}Data downloaded from the <a href="/funders/">funders</a> and <a href="/recipients/">recipients</a> pages are different from the search results and grants view downloads as they contain only the details seen on screen, plus an extra column
+    with the funder or recipient identifiers.{% endblocktrans %}</p>
+
+  <div class="row bottom-space">
+    <div class="col-xs-12">
+      <table class="table table-condensed table-bordered table-striped dt-responsive" width="100%">
+        <tr>
+          <th>{% trans "Field" %}</th>
+          <th>{% trans "Description" %}</th>
+        </tr>
+        <tr>
+          <td>{% trans "Funder / Recipient" %}</td>
+          <td>{% trans "Funder or recipient name" %}</td>
+        </tr>
+        <tr>
+          <td>{% trans "Funder Id / Recipient Id" %}</td>
+          <td>{% trans "Identifier code of the funder or recipient" %}</td>
+        </tr>
+        <tr>
+          <td>{% trans "Grants" %}</td>
+          <td>{% trans "Number of grants made by/to the funder or recipient" %}</td>
+        </tr>
+        <tr>
+          <td>{% trans "Total" %}</td>
+          <td>{% trans "Total value of all grants made by/to the funder or recipient" %}</td>
+        </tr>
+        <tr>
+          <td>{% trans "Average" %}</td>
+          <td>{% trans "Average value of a grant made by/to the funder or recipient" %}</td>
+        </tr>
+        <tr>
+          <td>{% trans "Largest" %}</td>
+          <td>{% trans "Value of the largest grant made by/to the funder or recipient" %}</td>
+        </tr>
+        <tr>
+          <td>{% trans "Smallest" %}</td>
+          <td>{% trans "Value of the smallest grant made by/to the funder or recipient" %}</td>
+        </tr>
+      </table>
+    </div>
+  </div>
+
+  <h2 id="advanced_search">{% trans "Advanced search" %}</h2>
+
+  <p>{% blocktrans %}In addition to the freetext search, there are many ways you can search the data on this site and target your search with more specific controls.{% endblocktrans %}</p>
+
+  <h3 id="words_phrases">{% trans "Words and phrases" %}</h3>
+  <ul>
+    <li>{% trans "A single word: e.g. <code>people</code>" %}</li>
+    <li>{% trans "Multiple words: e.g. <code>young people gardens</code> (each word does not necessarily have to be present)" %}</li>
```

Comment:
What does that mean?

Reply:
[At the moment(http://grantnav.threesixtygiving.org/advanced_search), this line of the advanced search help says
> All words: e.g. young people gardens

I didn't think that very clear, because it implies all words will be present in wht is found when this isn't the case. 

This would search for grants which contain these words, but it will return results that contain only "young" and "people", but not "gardens", for example. 

Technically, if you wanted to find **all words** you would use `young AND people AND gardens` which the next line outlines